src/dbally/views/structured.py

```diff
@@ -94,6 +107,23 @@ async def apply_filters(self, filters: IQLQuery) -> None:
             filters: [IQLQuery](../../concepts/iql.md) object representing the filters to apply
         """
 
+    @abc.abstractmethod
+    def list_aggregations(self) -> List[ExposedFunction]:
+        """
+
+        Returns:
+            Aggregations defined inside the View.
+        """
+
+    @abc.abstractmethod
+    async def apply_aggregation(self, aggregation: IQLQuery) -> None:
```

Comment:
Aggregation can also be a single function call. We don't support IQL for aggregations. So this argument should be typed accordingly.